app/src/main/java/com/github/se/gatherspot/model/Category.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.se.gatherspot.model
+
+enum class Category {
+  SPORT,
+  ARTS,
+  GAMES,
+  CONCERT,
+  MUSIC,
+  DANCE,
+  BASKETBALL,
+  FOOTBALL,
+  GAMING,
+  // TODO: Replace this with real categories
+  DEFAULT_CATEGORY,
+  DEFAULT_CATEGORY_1,
+  DEFAULT_CATEGORY_2,
+  DEFAULT_CATEGORY_3,
+  DEFAULT_CATEGORY_4,
+  DEFAULT_CATEGORY_5,
+  DEFAULT_CATEGORY_6,
+  DEFAULT_CATEGORY_7,
+  DEFAULT_CATEGORY_8,
+  DEFAULT_CATEGORY_9,
+  DEFAULT_CATEGORY_10,
+  DEFAULT_CATEGORY_11,
+  DEFAULT_CATEGORY_12,
+  DEFAULT_CATEGORY_13
```

Comment:
Replace these from the list we sent (or make sure to not use it when Melvin's class get merged)